service-mvt/src/main/java/org/oskari/service/mvt/WFSTileGrid.java

```diff
@@ -17,15 +18,28 @@ public WFSTileGrid(double[] extent, int maxZoom) {
 
         this.originX = extent[0];
         this.originY = extent[3];
+        this.origin = new double[]{this.originX, this.originY};
 
         this.resolutions = new double[maxZoom + 1];
 
         resolutions[0] = w / TILE_SIZE;
-        for (int i = 1; i < maxZoom; i++) {
+        for (int i = 1; i <= maxZoom; i++) {
```

Comment:
Fixes `z too high for layer`. Resolution array contained 0 as the last value of the array.